src/components/cylc/tree/TreeItem.vue

```diff
@@ -85,6 +85,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               />
             </div>
             <span class="mx-1">{{ node.name }}</span>
+            <v-chip
+              label
+              density="compact"
+              size="small"
+              class="ml-1 cursor-default"
+              :prepend-icon="$options.icons.flow"
+              :disabled="node.node.state === 'waiting' && !node.node.isQueued"
```

Comment:
N.B. this is for developer purposes only (showing dimmed flow nums for tasks in the data store that have not spawned yet, which could have different flow nums when they actually spawn). We should make this condition a `v-if` instead to hide data store flow nums.